source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -195,19 +264,52 @@ void MGETRequest::onChildResponse(Common::Redis::RespValuePtr&& value, uint32_t
   }
 }
 
+void MGETRequest::recreate(Common::Redis::RespValue& request, uint32_t index, bool prepend_asking) {
+  int num_values = prepend_asking ? 3 : 2;
```

Comment:
nit: unsigned type here. Also, can you add some comments or potentially use a constant w/ comments for all the magic uses of "3" and "2" throughout this change?

Reply:
done